galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/ServletErrorMessage.java

```diff
@@ -84,7 +84,11 @@ public enum ServletErrorMessage {
 
     // Auth APIs...
     GAL5051_INVALID_GALASA_TOKEN_PROVIDED             (5051, "E: Invalid GALASA_TOKEN value provided. Please ensure you have set the correct GALASA_TOKEN property for the targeted ecosystem at ''{0}'' and try again."),
-    GAL5052_FAILED_TO_RETRIEVE_CLIENT                 (5052, "E: Unable to retrieve client for authentication. Please ensure you have set the correct GALASA_TOKEN property for the targeted ecosystem at ''{0}'' and try again.")
+    GAL5052_FAILED_TO_RETRIEVE_CLIENT                 (5052, "E: Unable to retrieve client for authentication. Please ensure you have set the correct GALASA_TOKEN property for the targeted ecosystem at ''{0}'' and try again."),
+    GAL5053_FAILED_TO_RETRIEVE_TOKENS                 (5053, "E: Error retrieving tokens from the auth store. Report the problem to your Galasa Ecosystem owner."),
+    GAL5054_FAILED_TO_GET_CONNECTOR_URL               (5054, "E: Could not get the URL of an upstream identity provider to authenticate with. Report the problem to your Galasa Ecosystem owner."),
```

Comment:
Internal server error. The REST API server could not get the URL of the authentication provider (eg: github/oidc) from the dex Galasa component. The dex component of Galasa could be badly configured, or could be experiencing a temporary issue. Report the problem to your Galasa Ecosystem owner.

Reply:
Done in https://github.com/galasa-dev/framework/pull/546/commits/e69cef6b4144a0cf1e64dd35b27af1e642be9ca9